src/node/warnings.h

```diff
@@ -7,14 +7,14 @@
 #define BITCOIN_NODE_WARNINGS_H
 
 #include <sync.h>
+#include <util/translation.h>
```

Comment:
I don't think this is needed? Maybe these patches would be a good opportunity to cleanup the includes of this module.

Reply:
Mmh, not sure what iwyu was reporting here before, but it seems correct now. Please resolve :)